### PR TITLE
Add the vitals summary chart

### LINKED ISSUE #943: Add the Vitals timeseries chart
**Is your feature request related to a problem? Please describe.**

image

**Describe the solution you'd like**
Use the Semiotic package to add the timeseries chart.

This issue does not include the goal line or the 7 and 28 day change components.

**Describe alternatives you've considered**
N/A

**Additional context**
[In progress data shape described here](https://paper.dropbox.com/doc/Data-shape-for-vitals--BHd3ufKZFwyOToFScYvBVqpSAg-j4g9yhvl0VWN1g60vwsqF)

```

- Proposed view shape:
    { "summary" | "ftr_enrolement" | ... : {
      date: string; // YYYY-MM-DD
      value: number;
      7d_avg: number; // needs to be separate.
      // Cannot calculate the earliest 7 days of averages from this data alone
      parent_7d_avg?: number; // average for the parent entity for this metric
    }}[]
- Proposed metadata shape:
    { key: {
      goal_value: number;
      goal_name: string;
      7d_change: number;
      28d_change: number;
    }}
```
Mocks: https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=2022%3A4

### PR DESCRIPTION
**Pull request by daschi**
## Description of the change

This PR adds the `VitalsSummaryChart` bar chart and a placeholder for the vitals summary detail. 

This PR doesn't include (and I've created separate tickets for):
- The 7 day rolling trendline
- The goal line
- Switching data on click of the summary metric cards
- The 7 and 28 day change metrics

Screen Shot 2021-03-24 at 4 11 15 PM

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #943 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/models/types.ts [hunk 1]
```diff
@@ -71,3 +71,10 @@ export type PopulationProjectionTimeseriesRecord = {
   totalPopulationMax: number;
   totalPopulationMin: number;
 };
+
+export type VitalsTimeseriesRecord = {
```
Comment (jovergaag) [inline src/core/models/types.ts:75]: I think some of these are going to change but I'll make that change in my API PR

**Review by jovergaag — APPROVED** [review]
Looks amazing. 📊

File: src/core/CoreNavigation.scss [hunk 2]
```diff
@@ -58,7 +58,7 @@
       &:hover,
       &:focus {
         background-color: $core-dropdown-color;
-        box-shadow: inset 0px -1px 1px $box-shadow-color;
+        box-shadow: inset 0px -1px 1px $box-shadow-color-30;
```
Comment (Catacola) [inline src/core/CoreNavigation.scss:61]: We use this pattern of `inset 0px -1px 1px $box-shadow-color-30` a lot. Would it be worth going one step deeper down the rabbit hole and doing this?
```scss
$inset-shadow-30: inset 0px -1px 1px $box-shadow-color-30;
...
box-shadow: $inset-shadow-30;
```

**Review by Catacola — APPROVED** [review]
Looks slick!

File: src/core/CoreNavigation.scss (hunk 2 above)
Comment (daschi) [reply src/core/CoreNavigation.scss:61]: Sure let's try it and see how it goes!

**Pull request merged**